predictor_robojudge.ipynb

```diff
@@ -69,31 +55,14 @@
     "latest_df.sample(3)"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# Get NPIs"
```

Comment:
This NPIs business is a leftover from some previous refactoring. It's gone now, we don't need to worry about NPIs here anymore.